.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+          cache: 'gradle'
+
+      - name: Grant execute permission
+        run: chmod +x gradlew
+
+      - name: Build
+        run: ./gradlew build
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)